src/create-template-orders.js

```diff
@@ -18,32 +17,49 @@ const LAST_START_TIMESTAMP_CUSTOM_OBJECT_KEY =
 // to address all the write inconsistencies when writing to database
 const INCONSISTENCY_MS = 3 * 60 * 1000
 
-const stats = {
-  processedCheckoutOrders: 0,
-  createdTemplateOrders: 0,
-  failedCheckoutOrders: 0,
-  duplicatedTemplateOrderCreation: 0,
-}
+async function createTemplateOrders({
```

Comment:
Did you actually change sth in this file?

Reply:
No probably diff conflicts, now it's not seen.